timetracker/entries/views.py

```diff
@@ -8,3 +8,25 @@ def entries(request):
     return render(request, 'entries.html', {
         'entry_list': entry_list,
     })
+
+
+def clients(request):
+    clients = Client.objects.all()
+    return render(request, 'clients.html', {
+        'clients': clients,
+    })
+
+
+def projects(request):
+    projects = Project.objects.all()
+    return render(request, 'projects.html', {
+        'projects': projects,
+    })
+
+
+def client_summary(request):
+    projects = Project.objects.all()
```

Comment:
You don't use this variable in your template, so no need to query the database ;)